### PR TITLE
Fixed nuget feed from dotnet-corefx to dotnet-core; update Metadata package

### PR DESCRIPTION
**Pull request by conniey**
- Move dotnet-corefx myget feed to dotnet-core
- Update Microsoft.CodeAnalysis because of TypeInitializationException in previous version
- Update System.Reflection.Metadata since it does not exist on the dotnet-core feed

### CODE REVIEW

**Comment by conniey** [comment]
@twsouthwick , @mjrousos 

One test is failing because it expects:
`M:TestClass.Bar(System.SByte optmod System.Runtime.CompilerServices.IsConst reqmod System.Runtime.CompilerServices.IsSignUnspecifiedByte*)`
but the only close to it in the actual results is:
`M:TestClass.Bar(System.SByte reqmod System.Runtime.CompilerServices.IsSignUnspecifiedByte optmod System.Runtime.CompilerServices.IsConst*)`

Original IL: [modopt.il](https://github.com/Microsoft/dotnet-apiport/blob/master/tests/Microsoft.Fx.Portability.MetadataReader.Tests/Tests/modopt.il#L23)

Is it possible the expected result is wrong?

**Comment by twsouthwick** [comment]
I believe we got the expected result from CCI - may be another bug in the metadata reader

File: src/Microsoft.Fx.Portability.MetadataReader/MemberMetadataInfoTypeProvider.cs [hunk 1]
```diff
@@ -395,19 +396,50 @@ public MemberMetadataInfo GetArrayType(MemberMetadataInfo elementType, ArrayShap
...
         {
             var builder = new StringBuilder();
             builder.Append(unmodifiedType.Name);
 
-            foreach (var modifier in modifiers)
+            builder.Append(isRequired ? " reqmod " : " optmod ");
+
+            MemberMetadataInfo info = null;
+
+            switch (modifierTypeHandle.Kind)
```
Comment (twsouthwick) [inline src/Microsoft.Fx.Portability.MetadataReader/MemberMetadataInfoTypeProvider.cs]: Looks like the order in which this received is different than before

**Comment by conniey** [comment]
Reflector shows: 
`public unsafe void Bar(sbyte modreq(IsSignUnspecifiedByte) modopt(IsConst)* x);`
CCI shows: 
`System.Void TestClass.Bar(System.SByte optmod System.Runtime.CompilerServices.IsConst reqmod System.Runtime.CompilerServices.IsSignUnspecifiedByte*)`

Maybe its a bug in CCI?

@nguerrera, Do you happen to have any insights?

**Comment by twsouthwick** [comment]
@conniey Do we need to update the library at this point? Can we just get the feed update in so builds won't be broken?

**Comment by conniey** [comment]
Yes. Because the old packages don't exist on the new feed. ;(

Cheers,
Connie

----- Reply message -----

**Comment by twsouthwick** [comment]
Makes sense. I'm going to merge this in so we can get the builds working again. I have an issue open to track this now.

**Pull request merged**